README.md

```diff
@@ -4,7 +4,28 @@
 
 An opinionated Kubernetes distribution with a focus on long-term on-prem deployments
 
-## Contributing
+# Contributing
```

Comment:
This would create another `<h1>`, no? whilst we already have one for the title. So stick to `<h2>` for this level (and below as well).